src/core/mesh/qgsmeshtracerenderer.h

```diff
@@ -128,20 +129,28 @@ class QgsMeshVectorValueInterpolatorFromFace: public QgsMeshVectorValueInterpola
 /**
  * \ingroup core
  *
- * Abstract class used to handle information about stream field;
+ * Abstract class used to handle information about stream field
  *
  * \note not available in Python bindings
  * \since QGIS 3.12
  */
 class QgsMeshStreamField
 {
   public:
+    struct FieldData
+    {
+      double mag;
+      float time;
+      int incX;
```

Comment:
please do not use abbreviations... FORTRAN95 is not my favourite compiler nowadays :)